### PR TITLE
Add my methods Array map,select,each

### PR DESCRIPTION
**Pull request by Megonar**

### CODE REVIEW

File: HW02/Ramanovich_Dzmitry/hw02.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module MyMethod
```
Comment (B0RlS) [inline HW02/Ramanovich_Dzmitry/hw02.rb:3]: Not a very good name for the module, not informative. It should reflect what the module is doing.

Comment (B0RlS) [inline HW02/Ramanovich_Dzmitry/hw02.rb:3]: The file name must be the same as the module name

Comment (B0RlS) [inline HW02/Ramanovich_Dzmitry/hw02.rb:3]: Could you please fix rubocop errors?

Comment (Megonar) [reply HW02/Ramanovich_Dzmitry/hw02.rb:3]: If i fix rubocop errors.  Rubocop will fix "for" method to "each".

Comment (Megonar) [reply HW02/Ramanovich_Dzmitry/hw02.rb:3]: Done

Comment (Megonar) [reply HW02/Ramanovich_Dzmitry/hw02.rb:3]: Done

**Comment by B0RlS** [comment]
snake_case is used for filenames, and camel case is used for module / class names in the file

**Comment by Megonar** [comment]
Done

File: HW02/Ramanovich_Dzmitry/hw02.rb (hunk 1 above)
Comment (B0RlS) [reply HW02/Ramanovich_Dzmitry/hw02.rb]: You can fix all errors and return your implemented loop.

**Comment by vladhilko** [comment]
You need to run rubocop only inside your folder 😅 You've updated all other tasks 
Screenshot 2021-04-25 at 20 47 30

**Comment by Megonar** [comment]
Sorry. I am stupid! I am already fixing this problem)

**Pull request closed without merging**